python_transport/wirepas_gateway/transport_service.py

```diff
@@ -621,6 +772,7 @@ def main():
     parse.add_mqtt()
     parse.add_gateway_config()
     parse.add_filtering_config()
+    parse.add_black_hole_parameters()
```

Comment:
Same comment as above regarding the choice of name

Reply:
so add_backend_connection_loss_parameters()

and parameters are prefixed with bcl_ ?

Reply:
thumbs up for add_backend_connection_loss_parameters(). You could even simplify it further, such as connection_loss_settings() or connection_buffer_settings().


For the arguments, I would prefer a longer name. I think it is always easier to remember, such as *connection_* or *buffer_*